dist/linux/common/org.cryptomator.Cryptomator.tray-unlocked.svg

```diff
@@ -2,12 +2,15 @@
 	<defs>
 		<style id="current-color-scheme" type="text/css">
 			.ColorScheme-Text {
-			color:#232629;
+			color:#222222;
+			}
+			.ColorScheme-Highlight {
+			color:#49B04A;
```

Comment:
Same as with `#222222`, let's use a fallback value that is in line with our brand, e.g. `#49B04A`.